### PR TITLE
Laravel Podcast

### LINKED ISSUE #270: Add all Laravel Podcast Season Four episodes as resources

### PR DESCRIPTION
**Pull request by Gummibeer**
closes #270

### CODE REVIEW

File: database/seeders/modules/beginning-php.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            [
+                'name' => 'The Laravel Podcast - Intro to Composer, with Jordi Boggiano',
+                'url' => 'https://laravelpodcast.com/episodes/intro-to-composer-with-jordi-boggiano',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Packages, with Freek Van der Herten & Marcel Pociot',
+                'url' => 'https://laravelpodcast.com/episodes/packages-with-freek-van-der-herten-marcel-pociot',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```
Comment (TammyTee) [inline database/seeders/modules/beginning-php.php:120]: Proposal: This should be a part of a new intermediate module surrounding Package Creation
```suggestion
```

File: database/seeders/modules/laravel-basics.php [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+            [
+                'name' => 'The Laravel Podcast - Authorization and Authentication, with Joseph Silber',
+                'url' => 'https://laravelpodcast.com/episodes/authorization-and-authentication-with-joseph-silber',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Presets & Jetstream, with Michael Dyrynda',
+                'url' => 'https://laravelpodcast.com/episodes/presets-jetstream-with-michael-dyrynda',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```
Comment (TammyTee) [inline database/seeders/modules/laravel-basics.php:252]: Proposal: This should be a part of a new intermediate or advanced module surrounding Authentication/Authorization
```suggestion
```

File: database/seeders/modules/laravel-basics.php [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+            [
+                'name' => 'The Laravel Podcast - The Service Container, with Christoph Rumpel',
+                'url' => 'https://laravelpodcast.com/episodes/the-service-container-with-christoph-rumpel',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```
Comment (TammyTee) [inline database/seeders/modules/laravel-basics.php:257]: Proposal: This should be a part of a new intermediate or advanced module surrounding Laravel Core
```suggestion
```

File: database/seeders/modules/laravel-basics.php [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+            [
+                'name' => 'The Laravel Podcast - Security, with Rizqi Djamaluddin',
+                'url' => 'https://laravelpodcast.com/episodes/security-with-rizqi-djamaluddin',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Queues, with Mohamed Said',
+                'url' => 'https://laravelpodcast.com/episodes/queues-with-mohamed-said',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```
Comment (TammyTee) [inline database/seeders/modules/laravel-basics.php:287]: Proposal: This should be a part of a new intermediate or advanced module surrounding Laravel Core
```suggestion
```

**Review by TammyTee — COMMENTED** [review]
@mattstauffer - @darkboywonder and I looked through these and made a few proposals of new modules we believe some of these episodes would be better suited in. Let us know your thoughts, and if all is good we'll create a task to get these in.

**Comment by TammyTee** [comment]
The removed resources will be added as part of #377

**Review by darkboywonder — APPROVED** [review]

**Pull request merged**